### ISSUE TITLE
Saving a group node as a template fails to load in a different browser tab

### ISSUE THREAD

**Issue by DaveMayfield** [comfyanonymous/ComfyUI#2828]
1. Use "convert to group node" on several nodes
2. Save the group node as a template
3. Open a new browser window pointing to ComfyUI
4. Do RightClick->NodeTemplates->TemplateName

Nothing happens. If I do the same thing in the original workflow it functions correctly. If I duplicate the original browser tab in Chrome it works correctly. But for a brand new tab it always fails. Other templates work fine, only group nodes fail in this way.

The combination of "convert to group node" and the ability to save it as a template will be game changer once we can re-use them in other workflows and browser tabs.

**Comment by AlterMann3** [comment]
I encountered the same problem. I want to save "group node"s as "component" and create a global "template" from it, but the processes are very complicated. I did some of them and couldn't do others, I couldn't figure out how I did... When I try to load the template, nothing appears on the screen, but new nodes are added to the _Add Node->group nodes->workflow_ section.

I can create and save "component"s, but I can't edit defaults this way, so I want to make a "template", but I couldn't figure it out.

**Comment by aprisma2008** [comment]
Yes its true. You can save it but cannot get it back. Its not in the menue list any more. Please fix that. Thank you so much
